### PR TITLE
medipens can be used in hand while resting on the floor

### PR DESCRIPTION
**Pull request by Tlaltecuhtli**
:cl:
fix: medipens can be used in hand while resting on the floor
/:cl:
you can click yourself but you cant use the shortcut because the can use topic wasnt set right ( its not set right anywhere so there are probably other canuse checks that should work when resting but they dont)

### CODE REVIEW

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 1]
```diff
@@ -123,7 +123,7 @@
 		update_icon()
 
 /obj/item/reagent_containers/hypospray/medipen/attack_self(mob/user)
-	if(user.canUseTopic(src, BE_CLOSE, FALSE, NO_TK))
+	if(user.canUseTopic(src, BE_CLOSE, FALSE, NO_TK, TRUE))
```
Comment (ShizCalev) [inline code/modules/reagents/reagent_containers/hypospray.dm:126]: make a new define, FLOOR_OKAY

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/modules/reagents/reagent_containers/hypospray.dm (hunk 1 above)
Comment (Tlaltecuhtli) [reply code/modules/reagents/reagent_containers/hypospray.dm:126]: where do i put it

**Comment by ExcessiveUseOfCobblestone** [comment]
someone please turn this into flags

**Review by kriskog — DISMISSED** [review]

**Comment by kriskog** [comment]
Merge conflict

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 2]
```diff
@@ -1,283 +1,283 @@
-/obj/item/reagent_containers/hypospray
```
Comment (ShizCalev) [inline code/modules/reagents/reagent_containers/hypospray.dm:1]: uhm

Comment (Tlaltecuhtli) [reply code/modules/reagents/reagent_containers/hypospray.dm:1]: i text compared the master and my pr and this file has no changes (othern than the one i made)
i think its just a github bug

Comment (Rohesie) [reply code/modules/reagents/reagent_containers/hypospray.dm:1]: The repo EoL's are changed to LF
Check the line ending in VSC:
vhIAMqYNcS
If it says CRLF click and change it to LF, then commit and push.

Comment (Tlaltecuhtli) [reply code/modules/reagents/reagent_containers/hypospray.dm:1]: where is that?

Comment (Rohesie) [reply code/modules/reagents/reagent_containers/hypospray.dm:1]: bottom-right in VSC

**Pull request closed without merging**